### ISSUE TITLE
Augmented diffs: different way metadata in old/new when members were the only modification

### ISSUE THREAD

**Issue by mojodna** [drolbr/Overpass-API#461]
In [the augmented diff for 2018-02-14T22:50:00Z](http://overpass-api.de/api/augmented_diff?id=2853595), [node 35989826](https://www.openstreetmap.org/node/35989826) was modified:

```xml
<action type="modify">
<old>
  <node id="35989826" lat="44.8953030" lon="-117.0541410" version="2" timestamp="2009-10-16T23:58:36Z" changeset="2869705" uid="147510" user="woodpeck_fixbot"/>
</old>
<new>
  <node id="35989826" lat="44.8952219" lon="-117.0543781" version="3" timestamp="2018-02-14T22:50:36Z" changeset="56368019" uid="289005" user="Utible"/>
</new>
</action>
```

[Way 5187240](http://openstreetmap.org/way/5187240) was affected (as [node 35989826](https://www.openstreetmap.org/node/35989826) is a member), but it was otherwise unmodified. I would expect metadata from version 3 for both `<old>` and `<new>` versions, but version 2 metadata is used for `<old>`. **Tags**, however, are both from version 3 (`created_by` and `tiger:reviewed` were present in version 2 but are not included in the diff).

```xml
<action type="modify">
<old>
  <way id="5187240" version="2" timestamp="2010-03-21T10:51:38Z" changeset="4188582" uid="20587" user="balrog-kun">
    <bounds minlat="44.8942740" minlon="-117.0735070" maxlat="44.8954170" maxlon="-117.0541410"/>
    <nd ref="35989826" lat="44.8953030" lon="-117.0541410"/>
    <nd ref="36031930" lat="44.8942970" lon="-117.0564250"/>
    <nd ref="35994851" lat="44.8942740" lon="-117.0581940"/>
    <nd ref="35998108" lat="44.8944570" lon="-117.0590310"/>
    <nd ref="35988199" lat="44.8947080" lon="-117.0596420"/>
    <nd ref="35991545" lat="44.8953030" lon="-117.0601890"/>
    <nd ref="36007807" lat="44.8954170" lon="-117.0606070"/>
    <nd ref="36013899" lat="44.8954160" lon="-117.0612140"/>
    <nd ref="36020057" lat="44.8953940" lon="-117.0735070"/>
    <tag k="highway" v="residential"/>
    <tag k="name" v="Estes Hill Road"/>
    <tag k="tiger:cfcc" v="A41"/>
    <tag k="tiger:county" v="Baker, OR"/>
    <tag k="tiger:name_base" v="Estes Hill"/>
    <tag k="tiger:name_type" v="Rd"/>
    <tag k="tiger:separated" v="no"/>
    <tag k="tiger:source" v="tiger_import_dch_v0.6_20070808"/>
    <tag k="tiger:tlid" v="137660256:137660257"/>
  </way>
</old>
<new>
  <way id="5187240" version="3" timestamp="2012-09-28T17:55:36Z" changeset="13288130" uid="306096" user="joakimfors">
    <bounds minlat="44.8942740" minlon="-117.0735070" maxlat="44.8954170" maxlon="-117.0543781"/>
    <nd ref="35989826" lat="44.8952219" lon="-117.0543781"/>
    <nd ref="36031930" lat="44.8942970" lon="-117.0564250"/>
    <nd ref="35994851" lat="44.8942740" lon="-117.0581940"/>
    <nd ref="35998108" lat="44.8944570" lon="-117.0590310"/>
    <nd ref="35988199" lat="44.8947080" lon="-117.0596420"/>
    <nd ref="35991545" lat="44.8953030" lon="-117.0601890"/>
    <nd ref="36007807" lat="44.8954170" lon="-117.0606070"/>
    <nd ref="36013899" lat="44.8954160" lon="-117.0612140"/>
    <nd ref="36020057" lat="44.8953940" lon="-117.0735070"/>
    <tag k="highway" v="residential"/>
    <tag k="name" v="Estes Hill Road"/>
    <tag k="tiger:cfcc" v="A41"/>
    <tag k="tiger:county" v="Baker, OR"/>
    <tag k="tiger:name_base" v="Estes Hill"/>
    <tag k="tiger:name_type" v="Rd"/>
    <tag k="tiger:separated" v="no"/>
    <tag k="tiger:source" v="tiger_import_dch_v0.6_20070808"/>
    <tag k="tiger:tlid" v="137660256:137660257"/>
  </way>
</new>
</action>
```

**Comment by mmd-osm** [comment]
Yes, this looks strange:

```
[adiff:"2018-02-14T22:50:00Z","2018-02-14T22:51:00Z"];
way(5187240);
out meta geom;
```

**Comment by drolbr** [comment]
Thank you for reporting the bug.

It has been fixed together with #462 in ff15392fb711539241d9417028faf54dadf7ae82 and 5466a550a6fe069588135315137110b99ebcb849

**Comment by mojodna** [comment]
Awesome, thanks!!
